src/components/item/sharing/CsvInputParser.tsx

```diff
@@ -43,46 +46,46 @@ const CsvInputParser = ({ item }: Props): JSX.Element => {
   const { t: translateBuilder } = useBuilderTranslation();
   const { t: translateMessages } = useMessagesTranslation();
   const { t: translateCommon } = useCommonTranslation();
-  const { id: itemId, path: itemPath } = item;
+  // const { id: itemId, path: itemPath } = item;
+  const { id: itemId } = item;
   const [isOpen, setIsOpen] = useState(false);
   const {
-    mutate: share,
+    // mutate: share,
     isLoading,
     isSuccess,
     isError,
     data: results,
     error,
   } = mutations.useShareItem();
 
+  // const { mutate: postItem } = mutations.usePostItem();
   const openModal = () => {
     setIsOpen(true);
   };
+
+  const { openModal: openMoveModal, selId: idTemplate, cleanItemSel: cleanId } = useContext(SelectItemModalContext);
+
   const handleClose = () => {
+    if (cleanId)
+      cleanId();
     setIsOpen(false);
   };
 
+  const { data: itemObj } = useItem(idTemplate)
   const handleFileChange = (e: ChangeEvent<HTMLInputElement>) => {
```

Comment:
I think you should store the file in the browser but only send it when the user confirm the action.

Reply:
Sorry for the confusion, I am querying the name of the template folder here.